parl/remote/master.py

```diff
@@ -57,14 +57,35 @@ def __init__(self, port):
         logger.set_dir(os.path.expanduser('~/.parl_data/master/'))
         self.ctx = zmq.Context()
 
+        self.master_ip = get_ip_address()
         self.client_socket = self.ctx.socket(zmq.REP)
         self.client_socket.bind("tcp://*:{}".format(port))
         self.client_socket.linger = 0
         self.port = port
 
-        self.job_center = JobCenter()
-
+        self.job_center = JobCenter(self.master_ip)
+        self.cluster_monitor = {'workers': {}, 'clients': {}}
         self.master_is_alive = True
+        self.client_hostname = defaultdict(int)
+
+    def _get_status(self):
+        status = {'workers': [], 'clients': []}
+        master_idx = None
+        for idx, worker in enumerate(self.cluster_monitor['workers'].values()):
+            _worker = worker.copy()
+            _worker['load_time'] = list(worker['load_time'])
+            _worker['load_value'] = list(worker['load_value'])
+            _worker['worker_address'] = self.job_center.worker_hostname[
```

Comment:
We should not access variables of job_center directly, as it's a private attribute of the `job_center`.